src/main/resources/templates/c_plus_plus/test/testUtils/AbstractTest.py

```
@@ -0,0 +1,269 @@
+
+        if msg:
+            self.__markAsFailed(f"timeout ({msg})")
+        else:
+            self.__markAsFailed("timeout")
+
+    def __loadFileContent(self, filePath: str) -> str:
+        """
+        Returns the content of a file specified by filePath as string.
+        """
+        if path.exists(filePath) and path.isfile(filePath):
+            with open(filePath, "r") as file:
```

Ruff

Unnecessary open mode parameters

Remove open mode parameters

(UP015)